crates/fluvio-sc-schema/src/objects/list.rs

```diff
@@ -27,7 +27,7 @@ where
 
 impl Request for ObjectApiListRequest {
     const API_KEY: u16 = AdminPublicApiKey::List as u16;
-    const DEFAULT_API_VERSION: i16 = 8;
+    const DEFAULT_API_VERSION: i16 = 9;
```

Comment:
Probably we want to update this also for `watch.rs` so we don't have problems when we do the Update controller